msu/systems/mod_settings/abstract_setting.nut

```diff
@@ -87,7 +87,11 @@
 			{
 				::MSU.System.ModSettings.updateSettingInJS(this.getPanelID(), this.getID(), _value);
 			}
+
+			return true;
 		}
+
+		return _value == this.Value;
```

Comment:
What's the reason to not return false here?

Reply:
That's actually a good point, by the time it gets here this will always be false.

Reply:
Because I want to return true if the already present value is equal to the passed _value.